uaaextras/tests.py

```diff
@@ -546,17 +537,51 @@ def test_post_change_password_bad_too_simple(self, render_template, uaac, zxcvbn
             c.post(
                 '/change-password',
                 data={
-                    'old_password': 'fdsa',
-                    'new_password': 'asdf',
-                    'repeat_password': 'asdf',
+                    'old_password': 'correct horse battery staple',
+                    'new_password': 'correct horse battery staple',
+                    'repeat_password': 'correct horse battery staple',
                     '_csrf_token': 'bar',
                 }
             )
 
         render_template.assert_called_with('change_password.html')
-        flash.assert_called_with('not great')
         assert not uaac().change_password.called
 
+    @patch('uaaextras.webapp.flash')
+    @patch('uaaextras.webapp.UAAClient')
+    @patch('uaaextras.webapp.render_template')
+    def test_post_change_password_good(self, render_template, uaac, flash):
+        """If the password does not violate any of the bad password things,
+           it should be accepted.
+        """
+        render_template.return_value = 'template output'
+
+        uaac().decode_access_token.return_value = {'user_id': 'example'}
+
+        with app.test_client() as c:
+            with c.session_transaction() as sess:
+                sess['_csrf_token'] = 'bar'
+                sess['UAA_TOKEN'] = 'foo'
+
+            c.post(
+                '/change-password',
+                data={
+                    'old_password': 'fdsa',
+                    'new_password': 'correct horse battery staple',
+                    'repeat_password': 'correct horse battery staple',
+                    '_csrf_token': 'bar',
+                }
+            )
+
+        uaac().change_password.assert_called_with(
+            'example',
+            'fdsa',
+            'correct horse battery staple'
+        )
+        render_template.assert_called_with('password_changed.html')
+        flash.assert_not_called()
+        assert uaac().change_password.called
```

Comment:
This assertion is redundant with the `assert_called_with` above. We already asserted that the method was called with specific arguments, so we don't need to assert again that it was called.